client/src/apps/code-collabo/components/AppPageStructure.tsx

```diff
@@ -28,27 +35,48 @@ export default function PageStructure({ children }: { children: ReactNode }) {
       <style jsx global>{ types }</style>
       <style jsx global>{ main }</style>
 
-      {/* Sidebar */}
-      <div>
-        <div><b>CODE COLLABO Sidebar Logo</b> goes here</div>
-        <nav>
-          <span><b>Sidebar Nav Menu</b> goes here: </span>
-          <button>Overview</button>
-          <button>Projects</button>
-          <button>Careers</button>
-          <button>Donate</button>
-        </nav>
-      </div>
-
-      {/* Page Content */}
-      <div>
-        <header>
-          <h2 className='page-title'>{thisPage}</h2>
-        </header>
-        <main>
-          { children }
-        </main>
-      </div>
+      { !isMobile ? (
+        // Web
+        <div className='container'>
+          {/* Sidebar */}
+          <div className='sidebar'>
+            <div><b>CODE COLLABO Sidebar Logo</b> goes here</div>
+            <nav>
+              <span><b>Sidebar Nav Menu</b> goes here: </span>
+              <Link href='/code-collabo'>Overview</Link>
+              <Link href='/code-collabo/projects'>Projects</Link>
+              <Link href='/code-collabo/careers'>Careers</Link>
+              <Link href='/code-collabo/donate'>Donate</Link>
```

Comment:
@Ifycode Should I also not concern myself about styles the spacing(paddings, margin)?

Reply:
Yes @Akpjunior94 
Including the correct font size of the fonts of the elements.

Reply:
But remember we don't use paddings (left and right), and margin (left and right) on big or outer containers - because of the problems generally associated with it as explained on the call. That is why we mix width plus margin auto for that.

Reply:
I think I have used paddings left and right somewhere, not sure though. I will confirm and revert accordingly.

Reply:
Yes, I have added comments there for you in the other PR.